response_operations_ui/controllers/survey_controllers.py

```diff
@@ -100,3 +100,40 @@ def update_survey_details(survey_ref, short_name, long_name):
         raise ApiError(response)
 
     logger.debug('Successfully updated survey details', survey_ref=survey_ref)
+
+
+def get_legal_basis_list():
+    logger.debug('Retrieving legal basis list')
+    url = f'{app.config["SURVEY_URL"]}/legal-bases'
+    response = requests.get(url, auth=(app.config['SURVEY_USERNAME'], app.config['SURVEY_PASSWORD']))
```

Comment:
Might be better to put SURVEY_BASIC_AUTH = (SURVEY_USERNAME, SURVEY_PASSWORD) in the config.

Reply:
Tis done (I didn't know you could do that ... :-) )